include/panda_torque_mpc/common.h

```diff
@@ -51,6 +52,14 @@ namespace panda_torque_mpc {
         return pinocchio::buildReducedModel(model_pin_full, locked_joints_id, q0_full);
     }
 
+    inline boost::shared_ptr<pinocchio::GeometryModel> loadPandaGeometryModel(pinocchio::Model model_pin)
+    {
+        std::string urdf_path = ros::package::getPath("panda_torque_mpc") + "/urdf/robot.urdf";
```

Comment:
```suggestion
        const std::string urdf_path = ros::package::getPath("panda_torque_mpc") + "/urdf/robot.urdf";
```
The value of this variable is not supposed to change later on, so it is better to keep it as a constant